third_party/nimble/wscript

```diff
@@ -69,17 +63,61 @@ def build(bld):
         'MYNEWT_VAL_BLE_SVC_DIS_SERIAL_NUMBER_READ_PERM=0',
         'MYNEWT_VAL_BLE_SVC_DIS_FIRMWARE_REVISION_READ_PERM=0',
         'MYNEWT_VAL_BLE_SVC_DIS_SOFTWARE_REVISION_READ_PERM=0',
+        'MYNEWT_VAL_BLE_SVC_DIS_SYSTEM_ID_READ_PERM=0',
+        'MYNEWT_VAL_BLE_SVC_DIS_HARDWARE_REVISION_READ_PERM=0',
+        'MYNEWT_VAL_BLE_SVC_DIS_SYSTEM_ID_DEFAULT=NULL',
```

Comment:
What are these needed to fix?

Reply:
Can we generate a sysconfig for TI to bring the two approaches in line?

Reply:
Please check (not tested on HW yet)